### PR TITLE
add StatusPage footer to docs

### PR DESCRIPTION
**Pull request by pedropregueiro**
That's it, add that

### CODE REVIEW

File: source/javascripts/statuspage.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+var sp = new StatusPage.page({ page: 'yvbv3z7hj69n'});
+
+sp.summary({
+  success: function(data) {
+    // adds the text description to the dropdown
+    $('.color-description').text(data.status.description);
+    // appends the status indicator as a class name so we can use the right color for the status light thing
+    $('.color-dot').addClass(data.status.indicator);
+  }
+});
```
Comment (KristopherJordan) [inline source/javascripts/statuspage.js]: new line here?

File: source/stylesheets/_variables.scss [hunk 2]
```diff
@@ -103,3 +103,12 @@ $search-box-border-color: #666 !default;
...
     hyphens: auto;
 }
+
+
+////////////////////////////////////////////////////////////////////////////////
+// STATUSPAGE
+////////////////////////////////////////////////////////////////////////////////
+$red:#e74c3c;
+$orange:#e67e22;
+$yellow:#f1c40f;
```
Comment (KristopherJordan) [inline source/stylesheets/_variables.scss]: I think this colour should be a little less orange.. Like #f4d247 or something

Comment (pedropregueiro) [inline source/stylesheets/_variables.scss]: lol, I just used the colors from StatusPage.

**Comment by pedropregueiro** [comment]
fixed it @KristopherJordan ;)

**Pull request merged**